content/03.methods.md

```diff
@@ -83,19 +83,19 @@ We used GATK Haplotype caller.
-The input interval BED files for both panel and WXS data were padded by 100 bp on each side for all variant calling algorithm runs.
-The BED files for WGS were not padded for Mutect2 and Strelka2 runs, were padded by 300 bp on each side for Lancet, and by 100 bp on each side for VarDict.
+The input interval BED files for both panel and WXS data, derived from probes used to create the assays, were padded by 100 bp on each side for all variant calling algorithm runs.
+We utilized the non-padded BROAD Institute interval calling list `wgs_calling_regions.hg38.interval_list`, comprised of the full genome minus N bases, as the WGS input BED, unless otherwise noted below. 
 Strelka2 [@doi:10/gdwrp4] v2.9.3 was run using default parameters on human genome reference hg38, canonical chromosomes only (chr1-22, X,Y,M), as recommended by the authors.
 The final Strelka2 VCF was filtered for PASS variants.
 Mutect2 from GATK v4.1.1.0 was run following Broad best practices outlined from their Workflow Description Language (WDL) [@url:https://github.com/broadinstitute/gatk/blob/4.1.1.0/scripts/mutect2_wdl/mutect2.wdl].
 The final Mutect2 VCF was filtered for PASS variants. 
-Lancet [@doi:10.1038/s42003-018-0023-9] v1.0.7 [@url:https://github.com/nygenome/lancet] was run using default parameters, unless noted below.
-For input intervals to Lancet, a reference BED was created by using only the UTR, exome, and start/stop codon features of the GENCODE 31 reference.  
-Per recommendations by the New York Genome Center, the Lancet input intervals were augmented with PASS variant calls from Strelka2 and Mutect2 as validation.
-VarDictJava [@doi:10.1093/nar/gkw227] v1.58 [@url:https://github.com/AstraZeneca-NGS/VarDictJava] was run using the hg38 fasta reference with the same BED intervals used for Mutect2.  
+To manage memory issues, VarDictJava [@doi:10.1093/nar/gkw227] v1.58 [@url:https://github.com/AstraZeneca-NGS/VarDictJava] was run using 20K interval chunks of the input BED, padded by 100 bp on each side, such that if an INDEL occurred in between intervals, it would be captured.
 Parameters and filtering followed BCBIO standards except that variants with a variant allele frequency (VAF) >= 0.05 (instead of >= 0.10) were retained. 
 The 0.05 VAF increased the true positive rate for INDELs and decreased the false positive rate for SNVs when using VarDict in consensus calling. 
 The final VCF was filtered for PASS variants with TYPE=StronglySomatic.
+Lancet [@doi:10.1038/s42003-018-0023-9] v1.0.7 [@url:https://github.com/nygenome/lancet] was run using default parameters, unless noted below.
+For input intervals to Lancet, a reference BED was created by using only the UTR, exome, and start/stop codon features of the GENCODE 31 reference and padded by 300 bp on each side. 
+Per recommendations by the New York Genome Center [@doi:10.1101/623702v1], the Lancet input intervals were augmented with PASS variant calls from Strelka2 and Mutect2 as validation.
```

Comment:
```suggestion
Per recommendations by the New York Genome Center [@doi:10.1101/623702v1], the Lancet input intervals were augmented with PASS variant calls from Strelka2 and Mutect2 (also padded by 300 bp on each side) as validation.
```

Reply:
No, the strelka2 and mutect2 were not padded at all (that's why I wrote non-padded above, unless otherwise noted below - to only specify within lancet and vardict, the padded.

Reply:
v2 removed in upcoming commit

Reply:
This is a wording question.. I was interpreting this sentence to mean that the lancet intervals included everything in the previous sentence, plus 600 bp around every SNV called by strelka and mutect.

Reply:
ahh, hmm

Reply:
`Per recommendations by the New York Genome Center [@doi:10.1101/623702], the Lancet input intervals described above were augmented with PASS variant calls from Strelka2 and Mutect2 as validation.`?